TOC now visible on small screens

digikar99 opened the pull request:
Bootstrap helps in better fonts - therefore keeping it, otherwise, it's not used anywhere.

vindarel commented:
Excellent change, thank you !

The blue is different (look at the two index pages) and I find your new blue too flashy. Thoughts ?

digikar99 commented:
Okay, the new blue was due to bootstrap version change 3.3.7 -> 4. 

I had updated bootstrap to version 4, in the hopes of using their sidebar - but implemented it in native CSS as that was simpler, but left the bootstrap version updated. I have reverted this change, and the blue should be as before now.
